sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/Models/BlobChangeFeedEvent.cs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Text;
+
+namespace Azure.Storage.Blobs.ChangeFeed.Models
+{
+    /// <summary>
+    /// BlobChangeFeedEvent.
+    /// </summary>
+    public class BlobChangeFeedEvent
```

Comment:
I would call it BlobChangeEvent

Reply:
I think we should consider leaving this as `BlobChangeFeedEvent` because we've got the same schema for changes routed through EventGrid and we don't have a Track 2 API or name for server-side logging event info yet.  `BlobChangeFeedEvent` is a little verbose, but I don't want to take a good name that we might want for the main Blobs package someday.